client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx

```diff
@@ -51,21 +33,31 @@ const InvestigationsInfo = (props: Props): JSX.Element => {
             <CardContent>
                 <Grid container>
                     <Grid item container xs={12} alignItems='flex-end' justify='space-around'>
-                        <Grid item xs={3} className={classes.investigationsGraphContainer}>
-                            <InvestigationBarChart investigationsGraphData={investigationsGraphData} />
-                        </Grid>
-                        <Grid item container alignItems='flex-end' justify='space-around' spacing={1} xs={9}>
+                        <Grid item container alignItems='stretch' spacing={3} xs={12}>
                         {
-                            investigationsGraphData.map((InvestigationData: InvestigationChart , index) => (
-                                <InvestigationInfoButton
-                                    id={`info-button-${index}`}
-                                    key={`investigationInfoButton-${index}`}
-                                    amountOfInvestigations={InvestigationData.value}
-                                    text={InvestigationData.id}
-                                    style={{ backgroundColor: InvestigationData.color }}
-                                    onClick={() => onInfoButtonClick(statusToFilterConvertor[InvestigationData.id], InvestigationData.id)}
-                                />
-                            ))
+                            investigationsGraphData.map((InvestigationData: InvestigationChart , index) => {
+                                return  (
+                                    <>
+                                    <Grid item xs={3}>
+                                        <InvestigationInfoButton
+                                            id={`info-button-${index}`}
+                                            key={`investigationInfoButton-${index}`}
+                                            amountOfInvestigations={InvestigationData.value}
+                                            text={InvestigationData.id}
+                                            style={{ backgroundColor: InvestigationData.color }}
+                                            onClick={() => onInfoButtonClick(statusToFilterConvertor[InvestigationData.id], InvestigationData.id)}
+                                        />
+                                    </Grid>
+                                    {InvestigationData.space &&  Array.from(Array(InvestigationData.space))
+                                        .map((_,i) => {
```

Comment:
what is the _ ?

Reply:
its ignore - a blank constant, Array.from(Array(number)) returns an empy array with width 'number' and i want to use the index